widget/form.go

```diff
@@ -48,13 +54,13 @@ func (f *Form) ensureGrid() {
 }
 
 // Append adds a new row to the form, using the text as a label next to the specified Widget
-func (f *Form) Append(text string, widget fyne.CanvasObject) {
+func (f *Form) Append(text string, widget fyne.CanvasObject) *Form {
```

Comment:
Is the return of the form still needed? AFAIK we don’t use chaining elsewhere. And ATM I doubt the business value of it :). Last but not least: If it should stay, it has to be covered by tests.